webapp/channels/src/components/sidebar/channel_navigator/channel_navigator.tsx

```diff
@@ -18,26 +18,11 @@ import * as Utils from 'utils/utils';
 import ChannelFilter from '../channel_filter';
 
 export type Props = {
-    canGoForward: boolean;
-    canGoBack: boolean;
-    canJoinPublicChannel: boolean;
-    showMoreChannelsModal: () => void;
-    showCreateUserGroupModal: () => void;
-    invitePeopleModal: () => void;
-    showNewChannelModal: () => void;
-    showCreateCategoryModal: () => void;
-    handleOpenDirectMessagesModal: (e: Event) => void;
-    unreadFilterEnabled: boolean;
-    canCreateChannel: boolean;
     showUnreadsCategory: boolean;
     isQuickSwitcherOpen: boolean;
-    userGroupsEnabled: boolean;
-    canCreateCustomGroups: boolean;
     actions: {
         openModal: <P>(modalData: ModalData<P>) => void;
         closeModal: (modalId: string) => void;
-        goBack: () => void;
-        goForward: () => void;
```

Comment:
All this is legacy code that is no longer used. It was polluting this file, so I piggybacked the change here.